pkg/api/v1alpha1/tinkerbelltemplateconfig.go

```diff
@@ -18,7 +18,7 @@ const TinkerbellTemplateConfigKind = "TinkerbellTemplateConfig"
 type ActionOpt func(action *[]tinkerbell.Action)
 
 // NewDefaultTinkerbellTemplateConfigCreate returns a default TinkerbellTemplateConfig with the required Tasks and Actions
-func NewDefaultTinkerbellTemplateConfigCreate(name string, versionBundle v1alpha1.VersionsBundle) *TinkerbellTemplateConfig {
+func NewDefaultTinkerbellTemplateConfigCreate(name string, versionBundle v1alpha1.VersionsBundle, diskType string) *TinkerbellTemplateConfig {
```

Comment:
Could we change to using `disk` or `diskPath`. We aren't really describing a type of disk. Also a code comment everywhere this is accepted, given its just a string, would go a long way so we establish expectations.